executor/agents/agno/agno_agent.py

```
@@ -28,6 +28,10 @@
 from shared.status import TaskStatus
 from shared.telemetry.decorators import add_span_event, trace_async
 
+if TYPE_CHECKING:
+    from shared.models.execution import ExecutionRequest
+from shared.telemetry.decorators import add_span_event, trace_async
```

Ruff (0.15.2)

[error] Redefinition of unused `add_span_event` from line 29: `add_span_event` redefined here

Remove definition: `add_span_event`

(F811)